src/psbt.cpp

```diff
@@ -38,8 +42,23 @@ bool PartiallySignedTransaction::Merge(const PartiallySignedTransaction& psbt)
 
 bool PartiallySignedTransaction::IsSane() const
 {
-    for (PSBTInput input : inputs) {
+    for (unsigned int i = 0; i < inputs.size(); ++i) {
+        const PSBTInput& input = inputs[i];
         if (!input.IsSane()) return false;
+        // Check issuance is empty in CTxOut but set in input
```

Comment:
This comment is confusing. The "Check issuance is empty in CTxOut" happens later, but it is referred to as it it happens just here. I think better wording would be along the lines of "Check that issuance is empty in the input, because later we check that issuance is empty in CTxOut"

Reply:
Reworded